scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterParam.json

```diff
@@ -14,6 +14,9 @@
     "workspaceRegion": {
       "value": "<workspaceRegion>"
     },
+    "enableContainerLogV2": {
+      "value": true
```

Comment:
Since we are changing the default to ContainerLogV2 and we would need to update the Public docs to reflect this.